src/core/evaluator.js

```diff
@@ -1468,12 +1502,22 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
                   })
               );
               return;
-            case OPS.setStrokeColorSpace:
+            }
+            case OPS.setStrokeColorSpace: {
```

Comment:
The curly braces here and above are a bit inconsistent with the other cases and don't really seem necessary, but I can imagine it looks a bit better to group the code. I don't mind either way, but perhaps we might consider making it more consistent later.

Reply:
That was actually necessary of how scopes work with `switch`-statements, please see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#Block-scope_variables_within_switch_statements

Reply:
That was actually necessary because of how scopes work with `switch`-statements, please see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#Block-scope_variables_within_switch_statements